app/src/views/OperationalLearning/OperationalLearningMap/index.tsx

```diff
@@ -0,0 +1,254 @@
+import {
+    MapLayer,
+    MapSource,
+} from '@togglecorp/re-map';
+
+import BaseMap from '#components/domain/BaseMap';
+import Link from '#components/Link';
+import MapContainerWithDisclaimer from '#components/MapContainerWithDisclaimer';
+import MapPopup from '#components/MapPopup';
+import useCountry from '#hooks/domain/useCountry';
+import {
+    COLOR_BLUE,
+    COLOR_LIGHT_BLUE,
+} from '#utils/constants';
+import {
+    adminFillLayerOptions,
+    getPointCircleHaloPaint,
+} from '#utils/map';
+import { GoApiResponse } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type learningStatsResponse = GoApiResponse<'/api/v2/ops-learning/stats/'>;
+
+const sourceOptions: mapboxgl.GeoJSONSourceRaw = {
+    type: 'geojson',
+};
+
+interface CountryProperties {
+    country_id: number;
+    name: string;
+    operation_count: number;
+}
+interface ClickedPoint {
+    feature: GeoJSON.Feature<GeoJSON.Point, CountryProperties>;
+    lngLat: mapboxgl.LngLatLike;
+}
+interface Props {
+   className?: string;
+   learning: learningStatsResponse | undefined;
+}
+const LEARNINGS_LOW_COLOR = COLOR_LIGHT_BLUE;
+const LEARNINGS_HIGH_COLOR = COLOR_BLUE;
+
+function getMaxLearning(learning: learningStatsResponse) {
+    const learningData = learning?.learning_by_country.filter(
+        ({ count }) => isDefined(count),
+    );
+
+    const maxLearning = maxSafe(
+        learningData?.map(({ count }) => count),
+    );
+    return maxLearning;
+}
+
+function OperationalLearningMap(props: Props) {
+    const strings = useTranslation(i18n);
+    const {
+        className,
+        learning,
+    } = props;
+
+    const [
+        clickedPointProperties,
+        setClickedPointProperties,
+    ] = useState<ClickedPoint | undefined>();
+
+    const countryResponse = useCountry();
+    const countryCentroidGeoJson = useMemo(
+        (): GeoJSON.FeatureCollection<GeoJSON.Geometry> => {
+            const features = countryResponse
+                ?.map((country) => {
+                    const learningList = learning?.learning_by_country?.find(
+                        (item: { country_id: number; }) => item.country_id === country.id,
+                    );
+                    if (isNotDefined(learningList)) {
+                        return undefined;
+                    }
+                    const units = learningList.count;
+                    return {
+                        type: 'Feature' as const,
+                        geometry: country.centroid as {
+                        type: 'Point',
+                        coordinates: [0, 0],
+                    },
+                        properties: {
+                            country_id: country.id,
+                            name: country.name,
+                            operation_count: units,
+                        },
+                    };
+                })
+                .filter(isDefined) ?? [];
```

Comment:
Mapping should be done the other way around and use a dict/map to find elements on a loop.